### ISSUE TITLE
Volunteer

### ISSUE THREAD

**Issue by ryanbeymer** [ryanbeymer/PMI-DEP-Wordpress#1]

**Comment by jwalker5006** [comment]
Hey Ryan... wanted to check in.  I am wanted to for a while move off of joomla and onto wordpress for our chapter, but PreDep sync has always been my stopper. Thought about writing my own plugin, but stumpled upon your start. Let me know how I can help.

**Comment by ryanbeymer** [comment]
@jwalker5006 same boat, i've got limited php skills, definitely not enough to drag this across the finish line. 

I reached out to PMI a few weeks ago to get credentials for accessing a test data set and they went dark with no response.

**Comment by jwalker5006** [comment]
Well, lets work together and maybe we can pull this thing off. To me the
joomla and starchapter options are OK, but they are not what you can turn
word press into.

*Jay D. Walker*

Cell: 608 217-4486

**Comment by ryanbeymer** [comment]
Yeah, that is my thoughts, we aren't getting the value out of joomla for what the chapter is paying. I'd much rather put that money elsewhere.
